src/content/docs/en/guides/deploy/space.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Deploy your Astro Site to Space
+description: How to build an Astro site in Space.
+type: deploy
+i18nReady: true
+---
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Deta Space](https://deta.space/developers) is a personal computer that lives in the cloud — a ‘personal cloud’. You can build and run apps for yourself in your own "Space". You can publish the apps you've built, and they'll run for people all around the world. 
+
+You can also install apps published by other devs. You'll get a personal copy of the app in your own cloud, running on your infrastructure. All the data that all these apps save to your Space is for you: to explore, to extend, and to export, if you like.
+
+This guide includes step-by-step instructions for building sites in Space. You can build any type of Astro site in Space: either a static site or a server-rendered site.
+
+## Prerequisite
+
+To push an Astro site to Space, make sure you first:
+
+- Create a [Space account](https://deta.space)
+- Install the [Space CLI](https://deta.space/docs/en/basics/cli) and log in
+
+No matter the type of Astro site you are building, create a Space project **inside** the directory of your Astro project. Run the CLI and follow the instructions on the screen.    
+
+```bash
+space new
+```
+
+:::note
+The Space CLI will try to detect the right configuration for your app. Accept the suggested configuration and follow the instructions below on how to configure it for your Astro app.
```

Comment:
```suggestion
The Space CLI will try to detect the right configuration for your app. Accept the suggested configuration, then follow the instructions below for configuring your project.
```
It's not often I get to throw a YouTube link in a PR comment, but this is one of my all time faves and I feel like you'll take this in the spirit in which it was intended!  :joy: https://youtu.be/Jtq1EBMe1gQ?t=96

Reply:
```suggestion
The Space CLI will try to auto-detect the configuration for your app. Accept the suggested configuration and then follow the instructions below, depending on the type of Astro app you want to deploy.
```

Haha -- now that you mention it, there is three forms of "configure" there. Perhaps something like the above?

Reply:
Ahahahah @sarah11918 good illustration

Reply:
I'm down with @xeust's suggestion here!